From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Kampas <martin.kampas@jolla.com>
Date: Fri, 2 Sep 2022 09:15:18 +0200
Subject: [PATCH] fix compatibility with older sed

The '-E' option is just a recently added alias for the long time
available '--regexp-extended' option.
---
 functions.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/functions.sh b/functions.sh
index f971d8618cb61a45da61de39d816e6018ac06b29..9457aa3c64db588451d09139ae7ad297c8715e0e 100755
--- a/functions.sh
+++ b/functions.sh
@@ -404,7 +404,7 @@ function cmp_rpm_meta ()
       do
         : "${REPLY}"
         files+=( "${REPLY}" )
-      done < <(diff -U0 $file1 $file2 | sed -E -n -e '/^\+\//{s/^\+//;s/ [0-9a-f]+ [0-9]+$//;p}')
+      done < <(diff -U0 $file1 $file2 | sed --regexp-extended -n -e '/^\+\//{s/^\+//;s/ [0-9a-f]+ [0-9]+$//;p}')
     fi
 
     if test -n "$sh"; then
